projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -0,0 +1,91 @@
+@import 'mixins';
+
+:host {
+  width: 100%;
+  max-width: 240px;
+
+  .multi-select {
```

Comment:
any reason this is nested inside the host block rather than inside the `.multi-select` block below?